lm_human_preference_details/train_reward_jax.py

```diff
@@ -636,7 +649,7 @@ def train(args: Args):
         start_text=args.task.start_text,
         end_text=args.task.end_text,
     )
-    normalization_dataloader = DataLoader(normalization_dataset, batch_size=args.rollout_batch_size)
+    normalization_dataloader = DataLoader(normalization_dataset, batch_size=args.rollout_batch_size * len(local_devices))
```

Comment:
Here I think it should be just `batch_size=args.rollout_batch_size`, because `args.rollout_batch_size` is already the global batch size scaled by the number of devices: `args.rollout_batch_size = int(args.local_rollout_batch_size * len(local_devices) * args.world_size)` in line 588.

Reply:
Imagine we have 

* `rollout_batch_size = 16`
* `local_rollout_batch_size = 1`
* `len(devices) = 8` (8 GPUs)
* `world_size=2` (2 distributed training processes, 16 GPUs total)

Then the data loader should have a batch size of 8, which we then shard to 8 devices locally in each of the two processes.

Reply:
Oh sorry, it should be `batch_size=args.local_rollout_batch_size * len(local_devices)`

Reply:
I see, thanks!